site/content/en/docs/manual/advanced/organization.md

```diff
@@ -5,57 +5,128 @@ weight: 2
 description: 'Using organization in CVAT.'
 ---
 
+**Organization** is a feature for teams of several users
+who work together on projects and share tasks.
+
+Create an **Organization**, invite your team members, and assign
+roles to make the team work better on shared tasks.
+
+See:
+
+- [Personal workspace](#personal-workspace)
+- [Create new organization](#create-new-organization)
+  - [Switching between organizations](#switching-between-organizations)
+- [Organization page](#organization-page)
+  - [Invite members into organization](#invite-members-into-organization)
+  - [Delete organization](#delete-organization)
+
 ## Personal workspace
 
-Your `Personal workspace` will display the tasks and projects you've created.
+The account's default state is activated when no **Organization** is selected.
+
+It means that a personal account is used.
+
+![](/images/personal_account.jpg)
+
+## Create new organization
+
+To create an organization, do the following:
+
+1. Log in to the CVAT.
+2. On the top menu, click your **Username** > **Organization** > **+ Create**.
+
+   ![](/images/image233.jpg)
+
+3. Fill in the following fields and click **Submit**.
+
+   ![](/images/image234.jpg)
+
+<!--lint disable maximum-line-length-->
+
+| Field            | Description                                                         |
+| ---------------- | ------------------------------------------------------------------- |
+| **Short name**   | A name of the organization that will be displayed in the CVAT menu. |
+| **Full Name**    | Optional. Full name of the organization.                            |
+| **Description**  | Optional. Description of organization.                              |
+| **Email**        | Optional. Your email.                                               |
+| **Phone number** | Optional. Your phone number.                                        |
+| **Location**     | Optional. Organization address.                                     |
 
-## Create a new organization
+<!--lint enable maximum-line-length-->
 
-To create an organization, open the user menu, go to `Organization` and click `Create`.
+The created organization will be available
+at you **nickname** > **Organization**
 
-![](/images/image233.jpg)
+### Switching between organizations
 
-Fill in the required information to create your organization.
-You need to enter a `Short name` of the organization, which will be displayed in the menu.
-You can specify other fields: `Full Name`, `Description` and the organization contacts.
-Of them will be visible on the organization settings page.
+If you have more than one **Organization**,
+it is possible to switch between these **Organizations** at any given time.
 
-![](/images/image234.jpg)
+Follow these steps:
+
+1. In the top menu, select your **Username** > **Organization**.
+2. From the drop-down menu, under the **Personal space** section,
+   choose the desired **Organization**.
+
+![](/images/image233_1.jpg)
```

Comment:
Need to mention, that if there are a lot of organization, instead dropdown menu user sees **Switch organization** button. 

<img width="94" alt="image" src="https://github.com/opencv/cvat/assets/40690378/5210e88e-71e8-4df0-96ba-bfe722f55ad1">

And then organization selector:
<img width="269" alt="image" src="https://github.com/opencv/cvat/assets/40690378/c530de90-ba0d-4b59-a3ee-af55abcbb5bd">


Try to create more than 10 organizations to see that